test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -76,6 +76,11 @@
     ("aten::slow_conv_transpose2d_backward", datetime.date(2022, 1, 31)),
     ("aten::slow_conv_transpose3d", datetime.date(2022, 1, 31)),
     ("aten::slow_conv_transpose3d_backward", datetime.date(2022, 1, 31)),
+    ("aten::linalg_svd", datetime.date(2022, 1, 31)),
+    ("aten::linalg_svd_out", datetime.date(2022, 1, 31)),
+    ("aten::linalg_svdvals", datetime.date(2022, 1, 31)),
+    ("aten::linalg_svdvals_out", datetime.date(2022, 1, 31)),
```

Comment:
`aten::_svd_helper` should be in this list because it's removed.

I don't think you should include other functions here.

Reply:
Actually, it was complaining because I changed the name of the input from `input` to `A`. I used that name in the docs when writing them, and @mruberry said that it was fine to change the names of non-kwonly arguments. We didn't change them all after writing the docs because it was not high prio.